### PR TITLE
Replace speed dial with 3 buttons

### LINKED ISSUE #1515: Replace speed dial
With buttons underneath the wallet selector. 

image

### PR DESCRIPTION
**Pull request by luckysori**
Fixes #1515.
Fixes https://github.com/get10101/10101/issues/1678.

https://github.com/get10101/10101/assets/9418575/4820fb6e-8b7f-43c3-9aa3-19a9d3c80a9c

---

- Choose a speficic style for the new buttons that diverges from the `WalletTheme`. This is purely subjective, I'm just proposing something (temporary) here. I also went for no icons, since I didn't like any of the options readily available. _I'm happy to change everything related to the styles._

- Remove fund button since the now always-visible `Receive` button serves the same purpose.

### CODE REVIEW

File: mobile/lib/features/wallet/wallet_screen.dart
Comment (Restioson) [inline mobile/lib/features/wallet/wallet_screen.dart:62]: Do we want this done before merge?

**Review by Restioson — APPROVED** [review]
Concept ACK; LGTM but not sure if we want to push the styling further in this PR or wait for follow-up

File: mobile/lib/features/wallet/wallet_screen.dart
Comment (luckysori) [reply mobile/lib/features/wallet/wallet_screen.dart:62]: I think that will just route to the screen in your ticket: https://github.com/get10101/10101/issues/1676.

**Pull request merged**
